reviewio/cli.py

```diff
@@ -11,12 +11,13 @@
 
 def extract_reviewers(pull_request, extract_weight):
     sources = [
-        set([rev.user for rev in pull_request.get_reviews() if rev.state in ['APPROVED', 'REQUEST_CHANGES']]),
-        set([rev for rev in pull_request.get_review_requests()[0] if isinstance(rev, NamedUser.NamedUser)]),
-        set([rev for rev in pull_request.get_review_requests()[1] if isinstance(rev, NamedUser.NamedUser)])
```

Comment:
test fo reviewer being NamedUser instance is required - otherwise team reviews will cause exceptions: https://github.com/borzecki/reviewio/pull/3